### PR TITLE
OpenVPN: depends on libpam

### PR DESCRIPTION
**Pull request by Char-Aznable**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

### CODE REVIEW

File: Formula/openvpn.rb [hunk 1]
```diff
@@ -13,6 +13,8 @@ class Openvpn < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "iproute2mac" if OS.linux?
+  depends_on "linuxbrew/extra/linux-pam" if OS.linux?
```
Comment (iMichka) [inline Formula/openvpn.rb]:
```
unless OS.mac?
  depends_on "iproute2mac"
  depends_on "linuxbrew/extra/linux-pam"
end
```

And move it to the end of the dependency list please.

File: Formula/openvpn.rb [hunk 2]
```diff
@@ -41,12 +43,14 @@ def install
 
     ENV.prepend_path "PKG_CONFIG_PATH", vendor/"pkcs11-helper/lib/pkgconfig"
 
+    ENV["IPROUTE"] = "#{Formula["iproute2mac"].opt_bin}/ip"
```
Comment (iMichka) [inline Formula/openvpn.rb]: `ENV["IPROUTE"] = "#{Formula["iproute2mac"].opt_bin}/ip" unless OS.mac?`

File: Formula/openvpn.rb [hunk 3]
```diff
@@ -41,12 +43,14 @@ def install
...
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
                           "--disable-silent-rules",
                           "--with-crypto-library=openssl",
                           "--enable-pkcs11",
-                          "--prefix=#{prefix}"
+                          "--prefix=#{prefix}",
+                          "--enable-iproute2=yes"
```
Comment (iMichka) [inline Formula/openvpn.rb]: `*("--enable-iproute2=yes" unless OS.mac?)`

**Comment by Char-Aznable** [comment]
I don't want to waste me time on this any more

**Comment by iMichka** [comment]
No problem. Anybody can pick this up later. Thanks.

**Pull request closed without merging**